resources/js/Pages/Project/Message.vue

```diff
@@ -31,10 +31,10 @@
             ...
           </li>
           <li class="di">
-            <inertia-link :href="'/' + $page.props.auth.company.id + '/teams/ships'">Messages</inertia-link>
+            <inertia-link :href="'/' + $page.props.auth.company.id + '/projects/' + project.id + '/messages'">{{ project.name }}</inertia-link>
           </li>
           <li class="di">
-            Message details
+            Detail of a message
```

Comment:
It needs to be translated

Reply:
Hum no, I need to delete this file, it’s not used anymore. Thanks for reminding me.